proto/exocore/operator/v1/query.proto

```diff
@@ -313,4 +306,52 @@ service Query {
       get: "/exocore/operator/v1/opt_info/{operator_and_avs.operator_addr}/{operator_and_avs.avs_address}"
     };
   }
-}
\ No newline at end of file
+
+  // Validators queries all validators that match the given status.
+  // When called from another module, this query might consume a high amount of
+  // gas if the pagination field is incorrectly set.
+  rpc Validators(QueryValidatorsRequest) returns (QueryValidatorsResponse) {
+    option (cosmos.query.v1.module_query_safe) = true;
+    option (google.api.http).get = "/exocore/operator/v1/validators/{chain}";
+  }
+
+  // Validator queries validator info for given validator address.
+  rpc Validator(QueryValidatorRequest) returns (QueryValidatorResponse) {
+    option (cosmos.query.v1.module_query_safe) = true;
+    option (google.api.http).get = "/exocore/operator/v1/validator/{validator_addr}/{chain}";
+  }
+}
+// QueryValidatorsRequest is request type for Query/Validators RPC method.
+message QueryValidatorsRequest {
+  // chain is the id of the chain served by the operator. here chain_id is not used since the
+  // Linter complains about capitalization, which can be set with a gogoproto.custom_name but
+  // that is not compatible with google.api.http.get in the Query service below.
+  string chain = 1;
+  // pagination defines an optional pagination for the request.
+  cosmos.base.query.v1beta1.PageRequest pagination = 2;
+}
+// QueryValidatorsResponse is response type for the Query/Validators RPC method
+message QueryValidatorsResponse {
+  // validators contains all the queried validators.
+  repeated Validator validators = 1
+      [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
+
+  // pagination defines the pagination in the response.
+  cosmos.base.query.v1beta1.PageResponse pagination = 2;
+}
+
+// QueryValidatorRequest is response type for the Query/Validator RPC method
+message QueryValidatorRequest {
+  // validator_addr defines the validator address to query for.
+  string validator_addr = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
Please rename to `validator_acc_addr` to avoid confusion.

Reply:
fixed